docs/configuration.md

```diff
@@ -54,7 +54,7 @@ polly.configure({
 });
 ```
 
-## recordIfExpired
+## recordIfExpired - _deprecated! use expiryStrategy_
```

Comment:
I think we forgot to push the latest update to this file. This one breaks formatting a bit..